packages/lesswrong/components/posts/PostsItemIntroSequence.tsx

```diff
@@ -88,8 +88,12 @@ export const styles = (theme: ThemeType): JssStyles=> ({
     flexGrow: 1,
   },
   sequenceImage: {
+    // In an ideal world, this would be `height: "100%"`. This works fine in Chrome, but
+    // a bug in Firefox means that this breaks the `width: "auto"` on the image
+    // resulting in a bad mis-positioning. Hard-coding the height is a hack, but it's a
+    // hack that works. See https://stackoverflow.com/questions/46480358
+    height: 76,
     position: "absolute",
-    height: "100%",
```

Comment:
This makes the image no longer responsive, i.e. if the title wraps then the image is too short. I tried just removing the `width: auto` from the child and that didn't seem to change anything, so you can prob do that?